test/column/renderEditCell.test.tsx

```diff
@@ -100,15 +100,15 @@ describe('Editor', () => {
     render(<EditorTest gridRows={rows} />);
     await userEvent.click(getCellsAtRowIndex(0)[0]);
     expect(getCellsAtRowIndex(0)).toHaveLength(2);
-
     await scrollGrid({ scrollTop: 2000 });
     expect(getCellsAtRowIndex(0)).toHaveLength(1);
     expect(screen.queryByRole('spinbutton', { name: 'col1-editor' })).not.toBeInTheDocument();
+    expect(getRows()[1]).toHaveTextContent('5353');
     await userEvent.keyboard('123');
-    expect(screen.getByRole('spinbutton', { name: 'col1-editor' })).toHaveValue(1230);
-    const spy = vi.spyOn(window.HTMLElement.prototype, 'scrollIntoView');
-    await userEvent.keyboard('{enter}');
-    expect(spy).toHaveBeenCalled();
+    expect(getCellsAtRowIndex(0)).toHaveLength(2);
+    expect(getRows()[1]).toHaveTextContent('11');
+    const editor = screen.getByRole('spinbutton', { name: 'col1-editor' });
+    expect(editor).toHaveValue(1230);
```

Comment:
Not sure what the new checks are for

Reply:
`etRows()[1]` gets the second _rendered_ row, no? Not necessarily the true "second" row.

Reply:
May be we can use something like `toBeVisible` 🤔

Reply:
Maybe we can just check `grid.scrollTop`